lib/ansible/modules/packaging/language/composer.py

```diff
@@ -50,6 +50,12 @@
             - Composer arguments like required package, version and so on.
         required: false
         default: null
+    php_path:
```

Comment:
please call this `executable` for consistency used for in other modules. e.g. https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/packaging/language/pip.py#L117 or also https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/packaging/language/npm.py#L53. Optionally add an `php_path` as an alias.